crates/protogen/src/sqlexec/physical_plan.rs

```diff
@@ -292,6 +292,9 @@ pub struct InterleaveExec {}
 #[derive(Clone, PartialEq, Message)]
 pub struct RuntimeGroupExec {}
 
+#[derive(Clone, PartialEq, Message)]
+pub struct DataSourceMetricsExecAdapter {}
```

Comment:
Yeah, `DataSourceMetricsExecAdapter` and `RuntimeGroupExec` (and others) may not have a body since the only thing it cares about is the child execution plan, which is automatically serialized/deserialized for us.

This _would_ be generated for us if we decided to define these in protobuf instead of using tonic macros directly, but this way was the easiest way of making use of types datafusion already defines in their protobuf without copying it into our repo.